bundles/org.openhab.binding.tapocontrol/pom.xml

```diff
@@ -13,5 +13,4 @@
   <artifactId>org.openhab.binding.tapocontrol</artifactId>
 
   <name>openHAB Add-ons :: Bundles :: TapoControl Binding</name>
-
```

Comment:
you may undo this change

Reply:
It's impossible to check and judge all technical deteails and functions in a large repo like oh. 
So you can just look if code is clean and looks fine. So thanks you for doing this.

What tests do you mean? 
If you mean the /test/ folder this was still a storage for code i tried some functions (like different device-discovery methods).
So it was no usefull stuff there at any time.

Reply:
I was referring to unit tests which are part of the CI run (mvn verify).
https://www.openhab.org/docs/developer/tests.html